r/tests/testthat/test-dataset.R

```diff
@@ -1061,6 +1061,17 @@ test_that("Writing a dataset: Parquet->Parquet (default)", {
   )
 })
 
+test_that("Writing a dataset: Not specified format", {
+  skip_on_os("windows") # https://issues.apache.org/jira/browse/ARROW-9651
+  dst_dir <- make_temp_dir()
+  write_dataset(mtcars, dst_dir)
+  new_ds <- open_dataset(dst_dir)
```

Comment:
Should we assert that this has produced parquet files / the dataset as opened is parquet files?

Reply:
sure, changing now

Reply:
updated in adf82e4

Reply:
FWIW the default format for open_dataset() is parquet, so this test would have failed if it hadn't written parquet files.